docs/dckrman.sh

```diff
@@ -1,6 +1,37 @@
 #!/bin/sh
 for i in $@; do
     b=$(basename $i)
+    if [[ "$b" =~ "podman-auto" ]] || \
+	   [[ "$b" =~ "exists" ]] || \
+	   [[ "$b" =~ "mount" ]] || \
+	   [[ "$b" =~ "podman-completion" ]] || \
+	   [[ "$b" =~ "podman-container-checkpoint" ]] || \
+	   [[ "$b" =~ "podman-container-cleanup" ]] || \
+	   [[ "$b" =~ "podman-container-clone" ]] || \
+	   [[ "$b" =~ "podman-container-restore" ]] || \
+	   [[ "$b" =~ "podman-container-runlabel" ]] || \
+	   [[ "$b" =~ "podman-generate" ]] || \
+	   [[ "$b" =~ "podman-healthcheck" ]] || \
+	   [[ "$b" =~ "podman-image-diff" ]] || \
+	   [[ "$b" =~ "podman-image-scp" ]] || \
+	   [[ "$b" =~ "podman-image-sign" ]] || \
+	   [[ "$b" =~ "podman-image-trust" ]] || \
+	   [[ "$b" =~ "podman-image-tree" ]] || \
+	   [[ "$b" =~ "podman-init" ]] || \
+	   [[ "$b" =~ "podman-kube" ]] || \
+	   [[ "$b" =~ "podman-machine" ]] || \
+	   [[ "$b" =~ "podman-network-update" ]] || \
+	   [[ "$b" =~ "podman-pod" ]] || \
+	   [[ "$b" =~ "podman-system-connection" ]] || \
+	   [[ "$b" =~ "podman-system-migrate" ]] || \
+	   [[ "$b" =~ "podman-system-renumber" ]] || \
+	   [[ "$b" =~ "podman-volume-export" ]] || \
+	   [[ "$b" =~ "podman-volume-import" ]] || \
+	   [[ "$b" =~ "reload" ]] || \
+	   [[ "$b" =~ "unmount" ]] || \
+	   [[ "$b" =~ "unshate" ]]; then
```

Comment:
typo, I think: `s/ate/are/`.

I'm uncomfortable with all the duplicated syntactic-sugar and with the hardcoding, but can't come up with anything better right now.